### DISCUSSION TITLE
application pack dev in github sucks

### DISCUSSION THREAD

**Discussion by ghost** [community/community#144960] [GitHub Education]
### Hi everyone,

Recently in fegma i took my student pack-sub easily not because it has no good verification way ... but because multi option ex: if your camera bad then you have to select from your device ) and what the hack my profile github have relation with dev pack ... i like my github accout to be so private just less information as possible ... but github-dev pack team still send you have to complete your damm profile (hey guys the only thing is not in my github is my a-ss pic ... should i put it then to take my damm student-verified ? or at least make it easy to contact you directly for those who has application problem ... not just a useless website but useless option (even when i change my camera input it stills shows the front camera ... ) how github dev pack can be this hard to take I'm gadmm CS Student badji mokhtar Sidi-Amar Annaba Algeria university with students cars and paper of the portal too

**Comment by ben-g123** [comment]
@ghost Any more info, since I got it successfully by uploading a cut out screenshot of the student card on my school portal and saying it is a "Student ID".
